include/roboteam_ai/interface/widgets/TreeVisualizerWidget.h

```diff
@@ -30,6 +30,11 @@ class TreeVisualizerWidget : public QTreeWidget {
 public:
     explicit TreeVisualizerWidget(MainWindow * parent);
     void setHasCorrectTree(bool hasCorrectTree);
+
+    /**
+     * Prevent future linker errors for undefined runtime vtable lookup
+     */
+    virtual ~ThreeVisualizerWidget() = default; 
```

Comment:
typo

Reply:
Okay well I'm basically removing boost:: as a dependency right now as both std::variant and std::optional exist now, which is the only thing you use from boost, so I'll fix that aswel